### ISSUE TITLE
Complete Chapter 4 of User Guide "Properties and Bindings"

### ISSUE THREAD

**Issue by msgilligan** [groovyfx-project/groovyfx#45]
Has anyone started writing this? If so, please point me at the work in progress.

I'd like to contribute this section unless someone else is working on it.

**Comment by russel** [comment]
@msgilligan  As far as I am aware no-one is actively working, or has pending material, so please feel free to steam ahead. If it turns out there are other contributions, I will act as editor and merge. I am fairly confident though that there are none just now.

**Comment by deanriverson** [comment]
Agreed.  The docs are long overdue for attention.  Please feel free to work on whatever you'd like!

**Comment by msgilligan** [comment]
@russel So I should submit the docs PR (when ready) against the `version8` branch or `master`?

**Comment by russel** [comment]
@msgilligan Very good question. Hummm… If you are working on the version8 branch with JDK8 then submit the changes to that branch. I will then cherry-pick them onto master.

**Comment by msgilligan** [comment]
Sounds like a plan!
